language-design.md

```diff
@@ -17,7 +17,56 @@ For each quote, describe what it was about the quote that led you pick it.
 
 **Response**
 
-
+> It is one thing to have learned the syntax of a language and another to have 
+> grasped the underlying paradigm [Yang and Radkin, 2015]
+
+We agree with this quote from _C is Manly, Python is for "n00bs": How False
+Stereotypes Turn Into Technical "Truths"_ for the most part, but in the case
+of Python and C, it is more complicated. Python does not require the user to
+have as sophisticated of an understanding of underlying paradigms because
+the intent of Python is to be user-friendly. To use C, on the other hand,
+it is necessary for the user to have at least a basic understanding of 
+underlying principles. An example is passing by reference. In C, you must
+explicitly pass a reference to a variable, whereas in Python you simply
+pass a parameter. 
```

Comment:
I think this is a good point - the intent and design of the two languages are so different (although if you look you can see the similarities) that it's hard to make an accurate comparison here.